tasks/skylab/FastqProcessing.wdl

```diff
@@ -0,0 +1,92 @@
+version 1.0
+
+task FastqProcessing {
+  input {
+    Array[File] r1_fastq
+    Array[File] r2_fastq
+    Array[File]? i1_fastq
+    File whitelist
+    String chemistry
+    String sample_id
+
+    # runtime values
+    String docker = "quay.io/humancellatlas/fastq-process"
```

Comment:
Is it possible to somehow identify the Docker with tool and version (i.e. sctools_v?)?

Reply:
The proposed name is quay.io/humancellatlas/secondary-analysis-sctools:v0.3.10. However, at this point it is in a cycle and egg cycle: if want to create this then we have to create a release of v0.3.10  for which we need to merge the sctools updates to master. Also, it does not allow me to push any push container that I build in to "quay.io/humancellatlas/secondary-analysis-sctools:vx.x.x". On the other hand, if we put quay.io/humancellatlas/secondary-analysis-sctools:v0.3.10. without having this pushed then it would fail cicrlceci tests. Therefore, the solution would be to have the sctools merged.

Reply:
yes it is changed

Reply:
can we remove the redundant import?